myhpom/tests/test_signup_view.py

```diff
@@ -0,0 +1,60 @@
+from django.core.urlresolvers import reverse
```

Comment:
Since you're changing the general structure of how tests were done before this PR, migrate the remaining test that doesn't follow that pattern...test_views.py to test_next_steps_view.py?

Reply:
good change.